apps/system/js/task_manager.js

```diff
@@ -174,6 +177,10 @@
    */
   TaskManager.prototype.hide = function cs_hideCardSwitcher() {
     if (!this._active) {
```

Comment:
I know this is not your change, but we have a this.isActive() method which we should use here